docs/authentication/authentication.mdx

```diff
@@ -37,125 +36,143 @@ import AuthenticatorTypeTable from '../includes/_authenticator-type-table.mdx';
 
 import { Badge } from "@nextui-org/react";
 
-Currently the following authentication factors are supported and more are coming soon:  
+Currently the following authentication factors are supported and more are coming soon:
 
 - Universal Passkey
 
 - FIDO2 Multi-Device Passkey - <Badge color="primary" variant="flat" >Coming soon!</Badge>
 
 - Email OTP - <Badge color="primary" variant="flat" >Coming soon!</Badge>
 
-- Federated IDP - <Badge color="primary" variant="flat" >Coming soon!</Badge> 
- 
+- Federated IDP - <Badge color="primary" variant="flat" >Coming soon!</Badge>
 
+## Hosted Web
 
+The Hosted Web Authenticator is a constrained version of the Embedded SDK authenticator type. It requires less configuration and code, but it does require redirecting your users to a web-based authenticator hosted by Beyond Identity to utilize your application's passkeys on a user's device. However, the Hosted Web Authenticator can be configured with your own logo and brand colors. Beyond Identity's hosted web app handles passkey registration and authentication for you, including generating new passkeys, presenting users with authenticator choice options as needed, and validating passkey assertions. Your app simply needs to redirect to Beyond Identity's hosted web authenticator, and we do the rest. This option works well with any OIDC library.
```

Comment:
I've been thinking about this one haha. I think this might help direct them to create an application and give some guidance on OIDc without being a separate guide.

```suggestion
The Hosted Web Authenticator is a constrained version of the Embedded SDK authenticator type. It requires less configuration and code, but it does require redirecting your users to a web-based authenticator hosted by Beyond Identity. The web-based authenticator, however, can be configured with your own logo and brand colors. The Hosted Web Authenticator handles passkey registration and authentication for you, including generating new passkeys, presenting users with authenticator choice options as needed, and validating passkey assertions. Your app simply needs to redirect to Beyond Identity's hosted web authenticator, and we do the rest. To use the Hosted Web Authenticator, your application needs an OpenID Connect client using the Authorization Code flow. We recommend looking for one on the list of [certified OpenID Connect clients](https://openid.net/developers/certified/). To get started, [create an application](/docs/next/add-an-application). All the values you need for your OpenID Connect client can be found in your Beyond Identity console application’s External Protocol tab. 
```